src/app/GitUI/Editor/FileViewerInternal.cs

```diff
@@ -838,39 +841,46 @@ void SetActiveLineNum(int line)
                 }
             }
 
-            public void Restore(bool isDiff)
+            public void Restore(string? contentIdentification)
             {
-                if (_viewer.TotalNumberOfLines <= 1)
+                _currentIdentification = contentIdentification;
+                if (_viewer.TotalNumberOfLines <= 1 || string.IsNullOrEmpty(contentIdentification) || string.IsNullOrEmpty(_currentIdentification))
```

Comment:
redundant
```suggestion
                _currentIdentification = contentIdentification;
                if (_viewer.TotalNumberOfLines <= 1 || string.IsNullOrEmpty(_currentIdentification))
```

Reply:
`_currentIdentification` must be set to keep the name of the current content (file) so it is known in capture

Reply:
In Capture()
In first commit, this was set with an explicit call, needed still for tests.